### ISSUE TITLE
Season Standings

### ISSUE THREAD

**Issue by schuurthing** [ynnadkrap/balldontlie#12]
Would love to be able to query standings. We seem to have all the pieces: teams, standings, games with timestamps etc.

**Comment by schuurthing** [comment]
If not a season standings endpoint, it would be great to be able to get a team's record for a season

**Comment by awjdthornton** [comment]
Yes, getting a team's record would be great for a given season. Would be nice to distinguish between regular season and playoffs in the result set or as a query parameter.
